mondrian/src/main/java/mondrian/olap4j/MondrianOlap4jProperty.java

```diff
@@ -94,7 +94,11 @@ public Datatype getDatatype() {
         case TYPE_BOOLEAN:
             return Datatype.BOOLEAN;
         case TYPE_NUMERIC:
-            return Datatype.UNSIGNED_INTEGER;
```

Comment:
This is what makes the test cases fail and why I changed the FoodMart.xml file.
Just reverted FoodMart.xml.

Reply:
Yeah I noticed this and thought this looked wrong. I even included it in the changes I suggested on the Jira. Numeric to unsigned integer is wrong, because you loose the decimals and the sign. The tests will need to change.